strings/accents/triton_replacement.json

```diff
@@ -1,750 +1,21 @@
-		"shop": "schjop",
-		"shoulder": "skulder",
-		"shout": "skout",
-		"show": "vise",
-		"simple": "enple",
-		"since": "siden",
-		"sing": "synge",
-		"single": "singel",
-		"sir": "herre",
-		"sister": "søster",
-		"sit": "sitte",
-		"site": "sted",
-		"sky": "skoj",
-		"sleep": "sleppe",
-		"small": "liten",
-		"snow": "snø",
-		"so": "så",
-		"society": "samfoiety",
-		"soft": "myk",
-		"some": "sone",
-		"son": "sønn",
-		"soon": "snart",
-		"space": "røm",
-		"special": "spesiell",
-		"spell": "spalle",
-		"spend": "spende",
-		"stand": "stån",
-		"star": "stjar",
-		"start": "starte",
-		"state": "stat",
-		"stay": "bli",
-		"step": "skritt",
-		"still": "fortsatt",
-		"stone": "stein",
-		"stop": "stoppe",
-		"store": "lager",
-		"straight": "strett",
-		"strange": "strenig",
-		"street": "strett",
-		"strong": "sterk",
-		"study": "studer",
-		"success": "sukses",
-		"such": "slik",
-		"sudden": "sooden",
-		"suffer": "soffer",
-		"suggest": "soggest",
-		"summer": "sommer",
-		"support": "søpote",
-		"sure": "skure",
-		"surprise": "skprese",
-		"swim": "svimme",
-		"table": "bord",
-		"take": "taken",
-		"talk": "talke",
-		"tall": "tøll",
-		"taste": "tasten",
-		"tax": "taxen",
-		"tea": "te",
-		"teach": "undervise",
-		"tell": "telle",
-		"ten": "ti",
-		"than": "enn",
-		"thank": "takk",
-		"that": "den",
-		"the": "den",
-		"their": "deres",
-		"them": "dem",
-		"then": "da",
-		"there": "der",
-		"these": "disse",
-		"they": "de",
-		"thing": "ting",
-		"think": "tenke",
-		"this": "thens",
-		"throw": "kaste",
-		"to": "til",
-		"top": "topp",
-		"touch": "berøre",
-		"toward": "mot",
-		"town": "by",
-		"tree": "tre",
-		"trouble": "trøbbel",
-		"true": "sann",
-		"try": "prøv",
-		"turn": "snurrn",
-		"understand": "forstå",
-		"until": "til",
-		"up": "opp",
-		"upon": "opån",
-		"very": "veldig",
-		"voice": "stemme",
-		"wait": "vente",
-		"war": "krig",
-		"warm": "varm",
-		"was": "vars",
-		"wash": "vashe",
-		"water": "vater",
-		"way": "vei",
-		"we": "vee",
-		"weak": "svak",
-		"wealth": "wælth",
-		"weather": "væther",
-		"wet": "våt",
-		"what": "hvat",
-		"where": "hvor",
-		"while": "mens",
-		"white": "whvit",
-		"who": "hvem",
-		"whole": "hel",
-		"why": "hvorfor",
-		"wide": "enorme",
-		"wimp": "minnow",
-		"will": "vil",
-		"win": "vinne",
-		"wind": "vind",
-		"window": "vindu",
-		"wish": "ønske",
-		"with": "minth",
-		"within": "innenfor",
-		"without": "uthuten",
-		"wolf": "ulv",
-		"wonder": "undre",
-		"word": "ord",
-		"work": "werk",
-		"world": "verden",
-		"worry": "bekymre",
-		"worth": "verdth",
-		"would": "ville",
-		"write": "skrive",
-		"wrong": "feil",
-		"year": "yeår",
-		"yet": "jet",
-		"young": "tiddle",
-		"youngling": "tiddler",
-		"yours": "dins",
-		"zero": "null"
+		"caught": "hooked",
+		"captured": "netted",
+		"coward": "ssshell-hider",
+		"buddy": "guppy",
+		"dummy": "clam-brain",
+		"idiot": "bubble-head",
+		"fucker": "crab-kicker",
+		"fuck": "barnaclesss"
```

Comment:
Wouldn't this cause them to say barnaclesssssssss with 9 s total? Just to be sure

Reply:
No the accent doesn't get affected by tongue speech. I have to write it like this so it shows correctly. If I wanted to replace a word with an s in it, I would need to make it the three s on the input side too, otherwise it isn't changed.